include/window.h

```diff
@@ -0,0 +1,25 @@
+#ifndef POKEDIAMOND_WINDOW_H
+#define POKEDIAMOND_WINDOW_H
+
+struct Window
+{
+    u8 * unk_00;
+    u8 unk_04;
+    u8 unk_05;
+    u8 unk_06;
+    u8 width;
+    u8 height;
+    u8 unk_9;
+    u16 unk_A_0:15;
+    u16 unk_A_15:1;
+    void * unk_C;
+};
+
+extern void FillWindowPixelBuffer(struct Window *, u32);
```

Comment:
again idk if window.c has been decomped but i'm too lazy to find out, if it has then the externs are not necessary

Reply:
It has not